src/cobertura.rs

```diff
@@ -562,6 +567,23 @@ fn write_lines(writer: &mut Writer<Cursor<Vec<u8>>>, lines: &[Line]) {
         .unwrap();
 }
 
+fn format_condition_coverage(conditions: &[Condition]) -> String {
+    let conditions_hit: f64 = conditions.iter().map(|c| c.coverage).sum();
+    let num_conditions = conditions.len();
+    if num_conditions > 0 {
+        format!(
+            "{:.0}% ({:.0}/{})",
+            100.0 * conditions_hit / num_conditions as f64,
+            conditions_hit,
+            num_conditions
+        )
+    } else {
+        // What is the sensible string to put here,
+        // when a branch has no conditions?
```

Comment:
Maybe 100%?

Reply:
Sure, that makes as much sense as anything else I guess :smile: